subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BomSupportPluginsSmokeTest.groovy

```diff
@@ -29,6 +30,7 @@ class BomSupportPluginsSmokeTest extends AbstractSmokeTest {
     static springVersion = "5.0.8.RELEASE"
 
     @Unroll
+    @ToBeFixedForInstantExecution(because = "Task with Configuration field in ResolveTestFixture")
```

Comment:
❓ wasn't this comment about the `configuration` field now annotated?

Reply:
Yes, I missed that one when reworking the PR, you've been quicker than CI to spot it!
Now fixed